### PR TITLE
[Content] Initial Hog Farm Push

### PR DESCRIPTION
**Pull request by WeathermanTrent**
## Pull Request Contents 

1. New story - 'North Carolina Hog Farms' 
2. Removed old EPA datasets and directed their use from the GHGC STAC
3. Updated and standardized topic tags on all existing stories and datasets
4. Updated data visualization zoom extents in some existing stories where map zoom was not set
5. Added GeoHealth paper citation to HMA heat stress and AOD stories, and changed disclaimer on burn scar story to note a paper is in progress (and added thesis citation to burn scar - will replace with paper citation when published)
6. Minor prose edits in dataset landing pages to clean them up (bigger standardization effort will come later)
7. Clarified obscure dataset names
8. Used internal <Link> style for linking to stories from other data story pages
9. Standardized disclaimer for community contributed stories

### CODE REVIEW

**Comment by jerikac** [comment]
FYI - @siddharth0248 and I are going to be using this PR to update other content errors/changes that are already in production (see https://github.com/NASA-IMPACT/veda-config/issues/433). Please do not merge until we've closed this issue!

**Comment by siddharth0248** [comment]
@jerikac @acblackford in the preview we see **Topic and Topics** in the filter panel.

**Comment by jerikac** [comment]
Let me sort through and adjust.... user error. Assuming we're going with "**Topics"**

File: datasets/bangladesh-landcover-2001-2020.data.mdx [hunk 1]
```diff
@@ -9,9 +9,12 @@ media:
     name: USGS
     url: https://unsplash.com/photos/d59NHNtT_Ss
 taxonomy:
-  - name: Topics
+  - names: Topics
```
Comment (slesaad) [inline datasets/bangladesh-landcover-2001-2020.data.mdx:12]: Should be 'name'

File: .env [hunk 2]
```diff
@@ -8,10 +8,10 @@ APP_DESCRIPTION=Visualization, Exploration, and Data Analysis (VEDA)
 APP_CONTACT_EMAIL=email@example.org
 
 # Endpoint for the Tiler server. No trailing slash.
-API_RASTER_ENDPOINT='https://openveda.cloud/api/raster'
+API_RASTER_ENDPOINT='https://staging.openveda.cloud/api/raster'
 
 # Endpoint for the STAC server. No trailing slash.
-API_STAC_ENDPOINT='https://openveda.cloud/api/stac'
+API_STAC_ENDPOINT='https://staging.openveda.cloud/api/stac'
```
Comment (slesaad) [inline .env:14]: this is still staging? has the data not been pushed to prod yet? if it is, revert this back to production urls

File: datasets/epa-annual--cover.jpg
Comment (slesaad) [inline datasets/epa-annual--cover.jpg:1]: reduce file size based on these guidelines - https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/content/frontmatter/media.md

File: stories/urban-heating.stories.mdx [hunk 3]
```diff
@@ -13,8 +13,21 @@ taxonomy:
...
+      - Heat
+  - name: Source
+    values:
+      - Community Contributed
 ---
 
+<Block> 
+  <Prose>
+
+    ###### This story is part of a study conducted on both heat and pollution stress in the Houston Metropolitan Area. [The data story that higlights pollution stress can be found here.](https://www.earthdata.nasa.gov/dashboard/stories/houston-aod)
```
Comment (slesaad) [inline stories/urban-heating.stories.mdx:26]: for internal links use `<Link>` component
https://github.com/US-GHG-Center/veda-config-ghg/pull/572/commits/f91b4041e359d15daea54e78b2fa01f4190c7af6

File: stories/camp-fire-burn-scar.stories.mdx [hunk 4]
```diff
@@ -29,6 +33,7 @@ taxonomy:
   <Prose>
     ## Introduction: The 2018 Camp Fire
     <mark>🚧 This Discovery presents work in progress and not peer-reviewed results! 🚧</mark>
```
Comment (slesaad) [inline stories/camp-fire-burn-scar.stories.mdx:35]: should this be called "data stories" instead?

File: .env (hunk 2 above)
Comment (anayeaye) [reply .env:14]: I started a PR a while back for these but I think we need a bit more info to get to production https://github.com/NASA-IMPACT/veda-data/pull/120

Comment (anayeaye) [reply .env:14]: ^ this was actually a guess at a need when I opened it (right after deleting the old collections). Now it sounds like we want to move forward with these

**Review by j08lue — COMMENTED** [review]
@jerikac just confirmed

> that the NC-Hogs VEDA data story PR is ready to be merged into Prod (the NLCD PR is not a blocker)

NB: The PR changes a lot of files besides that one story. Mostly adding taxonomy to many existing datasets and stories and removing (or replacing?) EPA datasets. 

These changes are should be acknowledge in the PR description.

File: .env (hunk 2 above)
Comment (j08lue) [reply .env:14]: This PR can only be merged once all data is available from production data services and we switched the endpoint URLs back to those.

**Review by j08lue — CHANGES_REQUESTED** [review]

File: .env (hunk 2 above)
Comment (freitagb) [reply .env]: is there a status update on this @j08lue and @anayeaye?

File: datasets/cmip6-tas.data.mdx [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 id: combined_CMIP6_daily_GISS-E2-1-G_tas_kerchunk_DEMO
-name: 'CMIP6 Daily GISS-E2-1-G Near-Surface Air Temperature (demo subset)'
+name: 'CMIP6 Daily GISS-E2-1-G Near-Surface Air Temperature (demo subset; 1950-2014)'
```
Comment (freitagb) [inline datasets/cmip6-tas.data.mdx:3]: Does this still need to include `demo subset`? Couldn't we just say "Historical CMIP6..."?

File: datasets/CMIP-winter-median-pr.data.mdx [hunk 6]
```diff
@@ -11,7 +11,14 @@ media:
...
   - name: Topics
     values:
-      - EIS
+      - Precipitation
+      - Snow
+      - Climate
+      - Climate Model
+  - name: Source
+    values:
+      - NASA EIS
```
Comment (freitagb) [inline datasets/CMIP-winter-median-pr.data.mdx:20]: in the dataset overview pages we have inconsistencies where provider is included as a source. Does EIS want a tag for their group in the dataset config file?

File: datasets/cmip6-tas.data.mdx [hunk 7]
```diff
@@ -26,7 +31,7 @@ infoDescription: |
 layers:
   - id: combined_CMIP6_daily_GISS-E2-1-G_tas_kerchunk_DEMO
     stacCol: combined_CMIP6_daily_GISS-E2-1-G_tas_kerchunk_DEMO
-    name: CMIP6 Daily GISS-E2-1-G Near-Surface Air Temperature (demo subset)
+    name: CMIP6 Daily GISS-E2-1-G Near-Surface Air Temperature (demo subset; 1950-2014)
```
Comment (freitagb) [inline datasets/cmip6-tas.data.mdx:34]: same question here

File: datasets/cmip6-tas.data.mdx [hunk 8]
```diff
@@ -26,7 +31,7 @@ infoDescription: |
...
     type: zarr
     tileApiEndpoint: 'https://prod-titiler-xarray.delta-backend.com/tilejson.json'
```
Comment (freitagb) [inline datasets/cmip6-tas.data.mdx:36]: Depending on timeline for titiler-xarray deployment we'll need to update these endpoints.

File: datasets/cmip6-tas.data.mdx [hunk 9]
```diff
@@ -26,7 +31,7 @@ infoDescription: |
...
     description: "Historical (1950-2014) daily-mean near-surface (usually, 2 meter) air temperature in Kelvin."
```
Comment (freitagb) [inline datasets/cmip6-tas.data.mdx:37]: should specify model used here.

File: datasets/darnah-flood.data.mdx
Comment (freitagb) [inline datasets/darnah-flood.data.mdx:1]: We will have to revisit datasets like this once we have titiler x-array/CMR working.

File: datasets/lahaina-fire.data.mdx [hunk 10]
```diff
@@ -11,10 +11,14 @@ media:
...
     values:
-      - EIS
+      - Disasters
+      - Burn Severity
+      - Wildfire
   - name: Source
     values:
-      - UAH
+      - HLS
+      - Landsat
```
Comment (freitagb) [inline datasets/lahaina-fire.data.mdx:20]: Does this include a Landsat data layer? I thought this was just HLS and if so, we shouldn't add Landsat here.

File: datasets/lis.da.trend.data.mdx [hunk 11]
```diff
@@ -11,7 +11,15 @@ media:
 taxonomy:
   - name: Topics
     values:
```
Comment (freitagb) [inline datasets/lis.da.trend.data.mdx:13]: Who suggested these topics? They're all over the map in specificity.

File: datasets/mo_npp_vgpm.data.mdx [hunk 12]
```diff
@@ -19,6 +19,10 @@ taxonomy:
   - name: Topics
     values:
       - Water Quality
+  - name: Source
+    values:
+      - Oregon State University
```
Comment (freitagb) [inline datasets/mo_npp_vgpm.data.mdx:24]: Another provider in the source list.

File: datasets/nighttime-lights.data.mdx [hunk 13]
```diff
@@ -11,7 +11,11 @@ media:
 taxonomy:
   - name: Topics
     values:
-      - Covid 19
+      - COVID 19
+  - name: Source
+    values:
+      - Black Marble
```
Comment (freitagb) [inline datasets/nighttime-lights.data.mdx:17]: this isn't the best `source` need something better.

File: datasets/ps_blue_tarp_detections.ej.data.mdx [hunk 14]
```diff
@@ -12,6 +12,10 @@ taxonomy:
   - name: Topics
     values:
       - Environmental Justice
+      - Disasters
+  - name: Source
+    values:
+      - NASA IMPACT
```
Comment (freitagb) [inline datasets/ps_blue_tarp_detections.ej.data.mdx:18]: provider. Source is Planet

File: datasets/snow-projections-diff.data.mdx [hunk 15]
```diff
@@ -11,7 +11,14 @@ media:
 taxonomy:
   - name: Topics
     values:
```
Comment (freitagb) [inline datasets/snow-projections-diff.data.mdx:13]: where did topic values here come from. Much more specific that others which leads to inaccurate/confusing faceted search results.

**Review by freitagb — CHANGES_REQUESTED** [review]
There are other areas where my comments here apply but i didn't want to do a bunch of copy/pastes.

File: datasets/CMIP-winter-median-pr.data.mdx (hunk 6 above)
Comment (acblackford) [reply datasets/CMIP-winter-median-pr.data.mdx:20]: there are currently >20 datasets in the dashboard catalog with NASA EIS Values tags, so I will not change these until we confirm if they want a tag or not.

File: datasets/darnah-flood.data.mdx
Comment (acblackford) [reply datasets/darnah-flood.data.mdx:1]: What would need to be changed/updated?

File: datasets/lis.da.trend.data.mdx (hunk 11 above)
Comment (acblackford) [reply datasets/lis.da.trend.data.mdx:13]: I believe EIS has a list they required we include.

File: datasets/mo_npp_vgpm.data.mdx (hunk 12 above)
Comment (acblackford) [reply datasets/mo_npp_vgpm.data.mdx:24]: Who would the Source be for this dataset then?

File: datasets/nighttime-lights.data.mdx (hunk 13 above)
Comment (acblackford) [reply datasets/nighttime-lights.data.mdx:17]: What would you suggest? We struggled to find a good source tag for this.

**Review by j08lue — APPROVED** [review]
I followed the reviews and have the impression that all comments have been addressed. 👌

**Review by aboydnw — APPROVED** [review]

**Comment by aboydnw** [comment]
@acblackford if you're ready, go ahead and merge
@snmln @hanbyul-here let's include this in the release today if possible

**Comment by acblackford** [comment]
@snmln @hanbyul-here why was this not included in the release yesterday?

**Comment by aboydnw** [comment]
@acblackford we haven't released veda-config yet, we'll post a message in slack when we do.

**Pull request merged**